modules/msk/mutect1/main.nf

```diff
@@ -0,0 +1,63 @@
+process MUTECT1 {
+    tag "$meta.id"
+    label 'process_low'
+    shell '/bin/sh'
+
+   
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'ghcr.io/msk-access/test:1.1.5':
+        'ghcr.io/msk-access/test:1.1.5' }"
+
+    input:
+    tuple val(case_sample_name), val(control_sample_name), path(case_bam), path(control_bam), path(case_bai), path(control_bai)
+    tuple val(meta), path(bed_file), path(fasta_file), path(fasta_index_file), path(fasta_dict_file)
+
+    output:
+    
+    path("*.mutect.vcf"), emit: vcf
+    path("*.mutect.txt"), emit: standard_mutect_output
+    
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def args2 = task.ext.args2 ?: ''
+    def bed_file = bed_file ? "--intervals ${bed_file}" : ''
+
+    
+    """
+    java -Xmx28g -Xms256m -XX:-UseGCOverheadLimit -jar /opt/mutect/muTect-1.1.5.jar -T MuTect \
+        --input_file:normal ${control_bam} \
+        --input_file:tumor ${case_bam} \
+        --intervals ${bed_file} \
+        --normal_sample_name ${control_sample_name} \
+        --reference_sequence ${fasta_file} \
+        --tumor_sample_name ${case_sample_name} \
+        --out ${case_sample_name}.${control_sample_name}.mutect.txt \
+        --vcf ${case_sample_name}.${control_sample_name}.mutect.vcf
```

Comment:
`args` needs to be added to the end / beginning of this command, depending on what the tool requires: https://github.com/nf-core/modules/blob/master/modules/nf-core/abacas/main.nf#L28. If we don't needs `args2` it can be removed.

Reply:
a36ee1214bb9a15bfa6797c356d7dc7263f351dc